libs/core/dtos/anime.dto.ts

```diff
@@ -0,0 +1,44 @@
+import { TimeIntervalDto } from './time-interval.dto';
+
+/** Anime DTO. */
+export type AnimeDto = {
+
+	/** Id of the current anime. */
+	readonly id: number;
+
+	/** Anime creation date in format 2024-07-16T06:21:11.171641Z. */
```

Comment:
```suggestion
	/** 
	 * Anime creation date in format .
	 * @example '2024-07-16T06:21:11.171641Z'
	 */
```

Please use jsdoc tags where possible